sdk/tables/azure-data-tables/tests/test_table.py

```diff
@@ -36,7 +36,7 @@ class StorageTableTest(AzureTestCase, TableTestCase):
     def test_create_properties(self, tables_storage_account_name, tables_primary_storage_account_key):
         # # Arrange
         account_url = self.account_url(tables_storage_account_name, "table")
-        ts = self.create_client_from_credential(TableServiceClient, tables_primary_storage_account_key, endpoint=account_url)
+        ts = TableServiceClient(credential=tables_primary_storage_account_key, endpoint=account_url)
```

Comment:
If this passes I'd keep it for now, but you can also change it to `credential=tables_primary_storage_account_key`. Small nit